jsonapi.gemspec

```diff
@@ -0,0 +1,16 @@
+version = File.expand_path('../JSONAPI_VERSION', __FILE__).strip
```

Comment:
yeah, I was just looking at the way Rails did it: https://github.com/rails/rails/blob/master/version.rb

neat . :-)